config/src/main/java/io/confluent/common/config/ConfigUtils.java

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright 2017 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+package io.confluent.common.config;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+
+public class ConfigUtils {
+  private final static Logger log = LoggerFactory.getLogger(ConfigUtils.class);
+
+  public static Properties translateDeprecated(Properties props, String[][] keyArray) {
```

Comment:
Might want to document the args a bit. Not obvious at first glance how the `String[][] keyArray` is supposed to be used. In fact, I'm surprised this is a `String[][]` instead of some sort of map.